src/main/webapp/app/overview/course-overview.component.html

```diff
@@ -38,7 +38,10 @@
                         <div jhiTranslate="artemisApp.courseOverview.menu.communication">Communication</div>
                     </div>
                     <div *ngIf="course.postsEnabled" jhiOrionFilter [showInOrionWindow]="false" class="tab-item discussion" routerLink="messages" routerLinkActive="active">
-                        <div jhiTranslate="artemisApp.courseOverview.menu.messages">Messages</div>
+                        <div class="d-flex align-items-center">
+                            <div jhiTranslate="artemisApp.courseOverview.menu.messages" class="d-inline-block me-1">Messages</div>
+                            <span [hidden]="messagesRouteLoaded || !hasUnreadMessages"><fa-icon [icon]="faMessage" class="text-info d-inline-block shakeIcon mb-1"></fa-icon></span>
```

Comment:
Couldn't we just use a bootstrap badge like [here](https://getbootstrap.com/docs/5.3/components/badge/#positioned) for consistency reasons?

Reply:
We have it at the conversation level as well with faMessage icon thats why i used the same icon.
![conv](https://user-images.githubusercontent.com/26312757/214571849-68f9d221-4a09-46e0-80dc-be7f07e312b8.png)